### PR TITLE
Ugly Number With Recursion (Addition of Ugly Number Problem #1286)

### PR DESCRIPTION
**Pull request by ErzaTitania-2001**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

Yes

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
Addition of Ugly Number Problem #1286

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@ErzaTitania-2001 Do not mix `ArmStrongRecursion.java` and `UglyRecursion.java` programs in the same PR. You can make separate PR for `ArmStrongRecursion.java` program.

File: Java/cp/UglyRecursion.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+//finding nth ugly number using recursion
```
Comment (ErzaTitania-2001) [inline Java/cp/UglyRecursion.java:1]: I'm quite new to GitHub. Sorry for the troubles. 
Ugly Problem link : https://github.com/TesseractCoding/NeoAlgo/issues/1286
PDDI Problem link : https://github.com/TesseractCoding/NeoAlgo/issues/1217

Comment (ErzaTitania-2001) [reply Java/cp/UglyRecursion.java:1]: @iamrajiv @atarax665 when is this pull request going to be resolved? Should I open two new PRs for each code individually?

**Review by iamrajiv — APPROVED** [review]

**Comment by iamrajiv** [comment]
@all-contributors please add @ErzaTitania-2001 for code

**Pull request merged**